diffs/packetbeatdiff.diff

```diff
@@ -0,0 +1,2602 @@
+               Kibana endpoint
+              </a>
++             endif::["packetbeat"!="apm-server"]
+             </li>
+            </ul>
+           </div>
+@@ -233,7 +234,7 @@
+            config file:
+           </p>
+           <h4>
+-           <a id="_literal_enabled_literal_10">
++           <a id="_enabled_10">
+            </a>
+            <code class="literal">
+             enabled
+@@ -280,7 +281,7 @@
+            </div>
+           </div>
+           <h4>
+-           <a id="_literal_certificate_authorities_literal">
++           <a id="_certificate_authorities">
+            </a>
+            <code class="literal">
+             certificate_authorities
+@@ -338,7 +339,7 @@
+            is specified.
+           </p>
+           <h4>
+-           <a id="_literal_key_passphrase_literal">
++           <a id="_key_passphrase">
+            </a>
+            <code class="literal">
+             key_passphrase
+@@ -355,7 +356,7 @@
+            file.
+           </p>
+           <h4>
+-           <a id="_literal_supported_protocols_literal">
++           <a id="_supported_protocols">
+            </a>
+            <code class="literal">
+             supported_protocols
+@@ -395,7 +396,7 @@
+            .
+           </p>
+           <h4>
+-           <a id="_literal_verification_mode_literal">
++           <a id="_verification_mode">
+            </a>
+            <code class="literal">
+             verification_mode
+@@ -431,7 +432,7 @@
+            .
+           </p>
+           <h4>
+-           <a id="_literal_cipher_suites_literal">
++           <a id="_cipher_suites">
+            </a>
+            <code class="literal">
+             cipher_suites
+@@ -551,7 +552,7 @@
+            </ul>
+           </div>
+           <h4>
+-           <a id="_literal_curve_types_literal">
++           <a id="_curve_types">
+            </a>
+            <code class="literal">
+             curve_types
+@@ -583,7 +584,7 @@
+            </ul>
+           </div>
+           <h4>
+-           <a id="_literal_renegotiation_literal">
++           <a id="_renegotiation">
+            </a>
+            <code class="literal">
+             renegotiation
+html/en/beats/packetbeat/master/configuration-template.html
+--- /dev/fd/63
++++ html/en/beats/packetbeat/master/configuration-template.html
+@@ -236,7 +236,7 @@
+              </code>
+              . The Packetbeat version is always appended to the given name, so the final name is
+              <code class="literal">
+-              packetbeat-%{[agent.version]}
++              packetbeat-%\{[agent.version]\}
+              </code>
+              .
+             </dd>
+@@ -263,7 +263,7 @@
+               </code>
+               . The Packetbeat version is always included in the pattern, so the final pattern is
+               <code class="literal">
+-               packetbeat-%{[agent.version]}-*

```

Comment:
Fixed in https://github.com/elastic/beats/pull/13722/commits/def5e8ef969937918badbd5ddf624addb3f3536c